services/core/shared/core/compat/sdk_v5/accounts.js

```diff
@@ -215,7 +239,10 @@ const resolveDelegateInfo = async accounts => {
 
 				account.username = account.dpos.delegate.username;
 				account.balance = account.token.balance;
-				account.pomHeights = await standardizePomHeights(account.dpos.delegate.pomHeights);
+				account.pomHeights = account.dpos.delegate.pomHeights
+					.sort((a, b) => b - a)
+					.slice(0, 5)
```

Comment:
@MaximeGagnebin The slicing has been done as concluded in the cross-team agreement during epic specification sessions.

Reply:
You're totally right @sameersubudhi , I didn't see that this would keep the 5 most recent PoMs, somehow I thought it kept the 5 oldest ones.
All good, will resolve.